GETTING_STARTED.md

```diff
@@ -0,0 +1,415 @@
+# Getting started
+
+In this document you will find a step-by-step guide on how to get Eunomia running in a local Minikube cluster.
+
+Eunomia is watching GitOpsConfig Custom Resources. They and their handling by Eunomia will be discussed in general.
+
+You will run two examples:
+- Eunomia acting upon local GitOpsConfig changes,
+- Eunomia watching for changes in a remote repository and acting upon them.
+
+## Prerequisites
+
+In order to build Eunomia, you'll need the following:
+
+- [Git](https://git-scm.com/downloads)
+- [Go 1.12+](https://golang.org/dl/)
+- [Docker](https://docs.docker.com/install/)
+- [Operator SDK v0.8.1](https://github.com/operator-framework/operator-sdk/blob/v0.8.1/doc/user/install-operator-sdk.md)
+- Access to a Kubernetes cluster
+  - [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
+  - [Minikube](https://kubernetes.io/docs/setup/minikube/) (for local development and testing)
+
+You will also have to [set up the GOPATH](https://github.com/golang/go/wiki/SettingGOPATH) env variable (the instructions are for Go 1.13 but it's the same for Go 1.12).
+
+### Installing on a Mac
+
+All the components (except operator-sdk v0.8.1) can be easily installed via [Homebrew](https://brew.sh/) on a Mac:
+
+```shell
+brew install git
+brew install go@1.12
+brew install docker
+brew install minikube
+brew install kubectl
+```
+
+Unfortunately, operator-sdk in version 0.8.1 isn't available via brew so you have to install it from GitHub release as described [on this page](https://github.com/operator-framework/operator-sdk/blob/v0.8.1/doc/user/install-operator-sdk.md).
+
+### Checkout your fork
+
+You wil need your own Eunomia fork later in the second example. When pushing changes to your fork, you will observe Eunomia make adjustments based on your changed GitOpsConfig.
+
+Check out [the Eunomia repository](https://github.com/KohlsTechnology/eunomia).
+
+1. Create a [fork](https://help.github.com/en/articles/fork-a-repo) of it
+2. Create the directories and clone your fork:
+
+```
+mkdir -p ${GOPATH}/src/github.com/KohlsTechnology
+cd ${GOPATH}/src/github.com/KohlsTechnology
+git clone https://github.com/<YOUR FORK>/eunomia.git
+cd eunomia
+```
+
+## Deploy Eunomia to a Minikube cluster
+
+### Start a local Minikube cluster
+
+Minikube is a minimal Kubernetes cluster run in a virtual machine in Virtualbox.
+
+```shell
+minikube start
+```
+
+From now on your local Kubernetes client (kubectl) is configured to use your just started Minikube cluster.
+
+### Deploy images to Minikube Docker registry
+
+The Eunomia operator gets packaged as a container image - we need to build it (with some other template processor images which [are described here](./README.md#Template-Engine)) and send to Minikube Docker registry to be easily accessible from the cluster.
+
+To achieve this run the helper script (assuming you are in your local Eunomia repository dir):
+
+```shell
+./scripts/deploy-to-local.sh minikube
```

Comment:
Why not using official `latest` image of eunomia? This command is needed during development, or for bleeding-edge dev versions of eunomia; don't we want users to only use the latest full release of eunomia? who's the target of this getting started doc?